rllib/algorithms/dreamerv3/dreamerv3.py

```diff
@@ -612,7 +643,7 @@ def training_step(self) -> ResultDict:
                 # update.
                 with self._timers["critic_ema_update"]:
                     self.learner_group.additional_update(
-                        timestep=self._counters[NUM_ENV_STEPS_TRAINED],
+                        timestep=self._counters[NUM_ENV_STEPS_SAMPLED],
```

Comment:
Really important change. I think most algos should count by env steps. This is the most common metric used in the literature when reporting results over time for RL.